<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="addBlockPB">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>340</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="addTimeSpinBox">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>70</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add New Block</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>70</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Block Duration</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="addGreenRadioButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>130</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">colorButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="addRedRadioButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>160</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">colorButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="addDimRadioButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>130</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Low</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">intnsityButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="addBrightRadioButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>160</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>High</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">intnsityButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>70</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>s</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="addAirRadioButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>270</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Air</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">gasTypeButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="addOdorRadioButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>270</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Odor</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">gasTypeButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="addNoneRadioButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>270</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>No Flow</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">gasTypeButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="addNoneBrightRadioButton">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>150</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>None</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">intnsityButtonGroup</string>
   </attribute>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>120</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>210</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>370</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>40</y>
     <width>20</width>
     <height>341</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Program</string>
   </property>
  </widget>
  <widget class="QListWidget" name="programList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>661</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>390</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Duplicate Block(s)</string>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>420</y>
     <width>20</width>
     <height>181</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_11">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>590</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="addDupBlocksPB">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>560</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>450</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Block To Duplicate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>490</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Range To Duplicate: First</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>525</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Range To Duplicate: Last</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBlockPB">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>670</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Row</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>610</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Configuration</string>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>630</y>
     <width>20</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_15">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>730</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="arduinoCommText">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>650</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>650</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>COMM</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>690</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Directory</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runPB">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>710</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="dupBlockText">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>440</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="dupBlocksFirstText">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>480</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="dupBlocksLastText">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>520</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loadProgramPB">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>670</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Existing</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="addFlowDir2RadioButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>230</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Direction 2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="addFlowDir1RadioButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>230</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Direction 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveProgramPB">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>670</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Program</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pickSaveFolderPB">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>690</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Pick Save Folder</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>650</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Baud</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="arduinoBaudText">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>650</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="savePathLabel">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>710</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>310</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="recordingCheckBox">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>320</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Reording ON</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="pairingTrialCheckBox">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>330</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Pairing Tag to Folder</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="gasTypeButtonGroup"/>
  <buttongroup name="intnsityButtonGroup"/>
  <buttongroup name="colorButtonGroup"/>
 </buttongroups>
</ui>
